### ISSUE TITLE
Tag auto completion in searches

### ISSUE THREAD

**Issue by falematte** [orgzly-revived/orgzly-android-revived#238]
It is quite important while searching in your notes to use tags. A lot of people use org-tags-view all the time in emacs. This is the same in Orgzly but there is no tag auto-completion when writing the query. This is very problematic because we should remember all the tags that we used. Is it possible to add this feature?

**Comment by amberin** [comment]
I agree, that would be great. There is working completion when editing tags, if we could leverage that it would be great.
